### ISSUE TITLE
Drop Down Menu Scolling

### ISSUE THREAD

**Issue by agrack** [hackathonismb/iCn-JupyterNotebook-3D#8]
After loading in an iCn3D window into Jupyter Notebooks with the following commands,

view=icn3dpy.view(q='mmdbid=3k83')
view 

if I have the menu icon at the top of my screen, the drop down menus work well. However with larger menus like Analysis and Style, if the icon is not at the very top it will invert the menu and hide many of the menu options.

<img width="954" alt="2023-04-10" src="https://user-images.githubusercontent.com/120054080/230992832-14ea6b04-94a5-4a68-b69a-93247ea40dde.png">
<img width="956" alt="2023-04-10(2)" src="https://user-images.githubusercontent.com/120054080/230992838-1bd6da3f-85f3-41f5-b299-e914755d7963.png">

**Comment by jiywang3** [comment]
Yes, that's an issue. You will notice that the menu is actually the short version (File > Customize Menus > Simple Menus). We'll improve it later.
